source/screens/Home/ActionsPane/ActionsPane.scss

```diff
@@ -23,6 +23,10 @@
   cursor: pointer;
   width: 100px;
 
+  &:hover {
```

Comment:
Having in mind that we are started supporting sending and receiving, "Receive" should always have opacity 1, and "Send" should be with opacity 0.5, by default, and get opacity 1 once wallet is synchronized (`initialized` flag in `useWalletSyncProgress` hook)